changelog/60216.fixed

```diff
@@ -0,0 +1 @@
+Stop SSH from hanging if connection is lost. Also added args to customized grace period.
```

Comment:
**question** Should this be "customized" or "customize"?

In other words, are we talking about the customized grace period, or are we talking about the args that allow you to customize the grace period? In the latter case, would it be better to say "Also added `server_alive_interval` and `server_alive_count_max` args to customize grace period"?

That would give interested users a jumping off point when coming to look at the issue or looking at the docs.